chaoscenter/graphql/server/manifests/cluster/3b_agents_deployment.yaml

```diff
@@ -10,7 +10,6 @@ data:
   VERSION: #{VERSION}
   START_TIME: #{START_TIME}
   SKIP_SSL_VERIFY: "#{SKIP_SSL_VERIFY}"
-  CUSTOM_TLS_CERT: "#{CUSTOM_TLS_CERT}"
```

Comment:
Any specific reason for making this change?

Rest of the changes look good 🚀

Reply:
This committed file changes are not supposed to be here. It solves [the issue](https://github.com/litmuschaos/litmus/issues/3943), and I have raised a [PR](https://github.com/litmuschaos/litmus/pull/4263) for it.